urlc8dbc7ce6e_dlubarov_guava_tgz-pJ8-MainJ8/src/util/FileUtils.java

```diff
@@ -0,0 +1,24 @@
+package util;
+
+import java.io.*;
+
+public final class FileUtils {
+    private FileUtils() {}
+
+    public static byte[] readFile(File f) throws IOException {
+        InputStream in = new FileInputStream(f);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *in*. Currently, there are execution paths that do not contain closure statements, for example, when an if condition at line 14 evaluates to true. Ensure *in* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).